### PR TITLE
Display docker version

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

Display docker version in use on the host

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by frenck** [comment]
I wonder if we should show this all the time?
As in, this would mainly be helpful for Supervised installations, right?

**Comment by ludeeus** [comment]
True, I'll add a check to hide if hassos, in the future maybe we could make this red if the user was running an unsupported version of docker?
Or is that already logged in the SU logs?

File: hassio/src/hassio-main.ts [hunk 1]
```diff
@@ -169,12 +175,14 @@ class HassioMain extends ProvideHassLitMixin(HassRouterPage) {
       return;
     }
 
-    const [supervisorInfo, hostInfo, hassInfo] = await Promise.all([
+    const [supervisorInfo, hostInfo, hassInfo, hassioInfo] = await Promise.all([
```
Comment (balloob) [inline hassio/src/hassio-main.ts:178]: Now we're delaying loading all hassio information until it is fetched. I guess it's fine for this one.

I wonder if in a future PR we should change to a new format where we have a `hassio` object with `{ supervisor, hassio, host, hass }`

File: hassio/src/hassio-panel-router.ts [hunk 2]
```diff
@@ -26,6 +27,8 @@ class HassioPanelRouter extends HassRouterPage {
 
   @property({ attribute: false }) public supervisorInfo: HassioSupervisorInfo;
 
+  @property({ attribute: false }) public hassioInfo!: HassioInfo;
```
Comment (balloob) [inline hassio/src/hassio-panel-router.ts:30]: `{ attribute: false }` is the default: https://github.com/Polymer/lit-element/blob/master/src/lib/updating-element.ts#L438-L446

**Review by balloob — APPROVED** [review]

File: hassio/src/hassio-panel-router.ts (hunk 2 above)
Comment (bramkragten) [reply hassio/src/hassio-panel-router.ts:30]: It is because that freaking lit plugin complains about it...

Comment (balloob) [reply hassio/src/hassio-panel-router.ts:30]: Can we disable that rule and file a bug?

Comment (bramkragten) [reply hassio/src/hassio-panel-router.ts:30]: Reading the code you posted ^ `{ attribute: false }` would not add it to `observedAttributes` whereas leaving it out would add it to `observedAttributes` as `hassioinfo` with type `string`.

So it is not the same, now we would never set `hassioinfo` so no problem for us.

But it is not a bug from `ts-lit-plugin` as there would be no way to convert the value set to the attribute `hassioinfo` to be of the type `HassioInfo` as the default for an attribute is a `string`.

Comment (balloob) [reply hassio/src/hassio-panel-router.ts:30]: Oh, interesting.

**Pull request merged**